cfme/tests/services/test_myservice.py

```diff
@@ -170,6 +172,39 @@ def test_service_link(appliance, context, myservice, provider):
         assert new_view.wait_displayed()
 
 
+@pytest.mark.parametrize('context', [ViaUI])
```

Comment:
Why is this parametrized against only a single parameter? Consistency? Or do you plan to add another parameter in the future?

Reply:
Yes , plan to add viaSSUI in future